doc/architecture/reference_points_coordinate_systems.adoc

```diff
@@ -0,0 +1,26 @@
+= Reference points and coordinate systems
+
+With Open Simulation Interface, an object's position coordinates refer to the center of the object's bounding box.
+This reference point is not necessarily identical to the origin of the respective coordinate systems.
```

Comment:
"respective coordinate system" is a bit confusing

Reply:
Fixed. See https://github.com/OpenSimulationInterface/open-simulation-interface/pull/516 for current changes.